src/modules/core/components/PayoutsList/PayoutsList.tsx

```diff
@@ -28,7 +27,7 @@ interface Props {
   maxLines?: number;
 
   /** Payouts list containing all the payouts */
-  payouts: Payouts;
+  payouts: any[];
```

Comment:
Because i removed type  Payouts, as it was used by Task feature. But component stays for now (because maybe will be use in future 🤦‍♀️ . Im actually thinking of moving this type back

Reply:
No, don't. Leave it as `any`.

We can add proper types for this, once we actually need to use it.